docs/references/config/config.toml.md

```diff
@@ -0,0 +1,2055 @@
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+|                     | `""`                                            |
+
+In case `$CMTHOME` is unset, it defaults to `$HOME/.cometbft`.
+
+Configures the location of the Write Ahead Log (WAL) for the mempool. `""`  disables the WAL.
+
+### mempool.size
+Maximum number of transactions in the mempool.
+```toml
+size = 5000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+If the mempool is full, incoming transactions are dropped.
+
+The value `0` is undefined.
+
+### mempool.max_txs_bytes
+The maximum size of all transactions accepted in the mempool.
+```toml
+max_txs_bytes = 1073741824
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+This is the raw, total transaction size. Given 1MB transactions and a 5MB maximum transaction size, mempool will only
+accept five transactions.
+
+The default value is 1 Gibibytes (2^30 bytes).
+
+### mempool.cache_size
+Mempool internal cache size for invalid transactions.
+```toml
+cache_size = 10000
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+The mempool cache is an internal store for invalid transactions. Storing invalid transactions help in filtering incoming
+transactions: we can compare incoming transactions to known invalid transactions and filter them out without going
+through the process of validating the incoming transaction.
+
+### mempool.keep-invalid-txs-in-cache
+Invalid transactions might become valid in the future, hence they are regularly removed from the mempool cache of
+invalid transactions. Turning this setting on will keep them in the list of invalid transactions forever.
+```toml
+keep-invalid-txs-in-cache = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+Invalid transactions might become valid at a later time. The mempool cache clears invalid transactions regularly so the
+transactions can be re-validated.
+
+If this setting is set to `true`, the mempool cache will NOT remove the invalid transactions. It is useful in cases when
+invalidated transactions can never become valid again.
+
+This setting can be used by operators to lower the impact of some spam transactions: when a large number of duplicate
+spam transactions are noted on the network, temporarily turning this setting to `true` will filter out the duplicates
+quicker than validating each transaction one-by-one. It will also filter out transactions that are supposed to become
+valid at a later date, so it is not advised to keep this `true` for a long time on regular networks as it can lead to
+valid transactions failing.
```

Comment:
```suggestion
valid at a later date.
```
I strongly disagree with the sentence I'm proposing to remove here. Unless there is a strong reason, this value should always be set to `true`, to avoid easily spamming a node.
The value `false` is ill-defined in this setting, let me know if you want further details on why. I have proposed to remove this setting in the past.